src/sentry/api/endpoints/group_first_last_release.py

```diff
@@ -4,18 +4,22 @@
 from sentry.api.base import EnvironmentMixin
 from sentry.api.bases import GroupEndpoint
 from sentry.api.helpers.group_index import get_first_last_release
+from sentry.ratelimits.config import RateLimitConfig, RateLimitGroup
 from sentry.types.ratelimit import RateLimit, RateLimitCategory
 
 
 class GroupFirstLastReleaseEndpoint(GroupEndpoint, EnvironmentMixin):
     enforce_rate_limit = True
-    rate_limits = {
-        "GET": {
-            RateLimitCategory.IP: RateLimit(5, 1),
-            RateLimitCategory.USER: RateLimit(5, 1),
-            RateLimitCategory.ORGANIZATION: RateLimit(5, 1),
-        }
-    }
+    rate_limits = RateLimitConfig(
+        group=RateLimitGroup.releases,
```

Comment:
Even though this returns releases I kind of wonder if it's more `group` related, since we'll tend to call this in relation to loading group details.